x/circuit/module.go

```diff
@@ -162,8 +164,9 @@ type Inputs struct {
 type Outputs struct {
```

Comment:
While you are here, it should be called `ModuleOutputs` and `ModuleInputs` to be consistent with the rest.